### PR TITLE
ANDROID-9497 Fixing promote action

### PR DESCRIPTION
**Pull request by gmerinojimenez**
### :goal_net: What's the goal?
[ANDROID-9497](https://jira.tid.es/browse/ANDROID-9497) Promote action was not working on remote (it did in local :( ), but in remote the sed command is not working
### :construction: How do we do it?
Just hardcoding the staging id, as it is not seems to be dangerous and it doesn't seems to change (I couldn't find documentation about it)

### ☑️ Checks
- [x] I updated the documentation (readmes, wikis, etc). If no need to update documentation explain why.

### :test_tube: How can I test this?
- [x] [Action result](https://github.com/Telefonica/mistica-android/actions/runs/956105758)

### CODE REVIEW

**Review by JorgeMucientes — APPROVED** [review]

**Comment by JorgeMucientes** [comment]
No ticket associated?

**Comment by gmerinojimenez** [comment]
> No ticket associated?

Not sure,

> No ticket associated?

added :)

File: build.gradle [hunk 1]
```diff
@@ -17,10 +17,10 @@ buildscript {
 }
 
 apply plugin: 'io.codearte.nexus-staging'
+
 nexusStaging {
     packageGroup = "com.telefonica"
-    stagingProfileId = "com.telefonica"
-    stagingProfileId = System.getProperty("STAGING_PROFILE_ID")
+    stagingProfileId = "f7fe7699e57a"
```
Comment (manolovn) [inline build.gradle:23]: is safe to put this value here?

**Review by manolovn — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (gmerinojimenez) [reply build.gradle:23]: yes, there is no way to do anything without the authentication, and with it, this can be retrieved easily.

**Pull request merged**